### PR TITLE
Phonebook.JaegerByte

### PR DESCRIPTION
**Pull request by JaegerByte**
My version of the Phonebook. Any tips or feedback would be greatly appreciated.
Thank you for the opportunity!

### CODE REVIEW

**Comment by TheCSharpAcademy** [comment]
@VocalNight Any updates on this one? 🤓

**Comment by JaegerByte** [comment]
> @VocalNight Any updates on this one? 🤓

No updates from me :D

**Comment by VocalNight** [comment]
Uh, did the review i made didn't get posted? One second, sorry about that.

**Review by VocalNight — CHANGES_REQUESTED** [review]

File: Phonebook.JaegerByte/InputValidationService.cs [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+        {
+            return !string.IsNullOrEmpty(input) &&
+                   Char.IsAsciiLetterUpper(input[0]) &&
+                   !input.Any(char.IsDigit) &&
+                   input.Length <= 20;
+        }
+        public bool ValidatePhonenumber(string input)
+        {
+            return !string.IsNullOrEmpty(input) &&
+                    input.All(Char.IsDigit) &&
```
[the remark this answers is not in the document]
Comment (VocalNight) [reply Phonebook.JaegerByte/InputValidationService.cs:15]: @JaegerByte Can you confirm if you can see the review now? Sorry for the delay man.

**Comment by TheCSharpAcademy** [comment]
@JaegerByte Let us know if you need help 🤓

**Comment by TheCSharpAcademy** [comment]
@JaegerByte We're closing the PR for admin purposes. Feel free to reopen it and resubmit in the academy when you're ready. @VocalNight Let me know if you don't get the points, as this function is new 😊

**Pull request closed without merging**